DBMS/Assignment4/zipcode.sql

```diff
@@ -0,0 +1,54 @@
+    -- Creating Database zipcodeInfo
+    CREATE DATABASE zipcodeInfo;
+    USE zipcodeInfo;
+    
+    -- creating table Zipcode
+    CREATE TABLE zipcodeNumber(
+        zipcode int PRIMARY KEY AUTO_INCREMENT
+    );
+    -- Creating table cityInfo
+    CREATE TABLE cityInfo(
+        zipcode int PRIMARY KEY AUTO_INCREMENT,
+        cityName varchar(20),
+        FOREIGN KEY (zipcode)
+            REFERENCES zipcodeNumber (zipcode)
+    );
+    -- Creating table stateInfo
+    CREATE TABLE stateInfo(
+        zipcode int PRIMARY KEY,
```

Comment:
ACTION: Correct your state table structure it should have only two columns state_id and state_name.